Admin/AbstractAdmin.php

```diff
@@ -2845,6 +2852,22 @@ public function getDashboardActions()
     /**
      * {@inheritdoc}
      */
+    public function getSearchResultLink($object)
```

Comment:
My thought is that making things `final` by default can help use realize what people want to do with the library. When they open a PR to make something not `final`, we must think hard about their use case to see if the method in question really is the best extension point for their needs.